android/build.gradle

```diff
@@ -130,4 +130,6 @@ dependencies {
     implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0'
     implementation 'com.google.android.material:material:1.9.0'
     implementation "androidx.core:core-ktx:1.8.0"
+    implementation "androidx.lifecycle:lifecycle-viewmodel:2.5.0"
+    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.5.0"
```

Comment:
As I checked those deps right now I observed that `"androidx.lifecycle:lifecycle-viewmodel-ktx:2.5.0"` could be sufficient, but in my opinion we should ensure that both of these libraries should have the same version.